test/kdb/remote_open_test.cpp

```diff
@@ -185,6 +187,8 @@ class RemoteDBFixture
 
     void Resolve(const string& p_accession)
     {
+        putenv((char*)"NCBI_VDB_NO_CACHE_SDL_RESPONSE=1");
```

Comment:
I may be better to do this in the CMakeLists.txt file, like:
`set_tests_properties( <test> PROPERTIES ENVIRONMENT_MODIFICATION "NCBI_VDB_NO_CACHE_SDL_RESPONSE=set:1" )`

Reply:
Sometimes I run failed tests in debugger.
Here it is better to set this variable from the binary.